### PR TITLE
Fix for Issue #37

### PR DESCRIPTION
**Pull request by bpickard**
Fixes for issue #37. 

Removes seeded answers when seeded options are removed and re-indexes remaining answer indexes
Update to unit tests to reflect change

### CODE REVIEW

File: ubcpi/static/js/spec/ubcpi_edit_spec.js [hunk 1]
```diff
@@ -277,14 +283,29 @@ describe('UBCPI_Edit module', function () {
...
-                {answer:2, rationale:'rationale3'}
+                {answer:2, rationale:'rationale3'},
+                {answer:1, rationale:'rationale2'},
+                {answer:0, rationale:'rationale1'}
             ]);
         });
 
+        it('should delete seeds containing an option when delete_option is called', function(){
+            var num_seeds = mockConfig.data.seeds.length;
+            controller.delete_option(2);
```
Comment (xcompass) [inline ubcpi/static/js/spec/ubcpi_edit_spec.js]: Could you change this to delete option 1 or add another test to test the seed answer re-indexing?

**Comment by xcompass** [comment]
Looks good :+1:  Could you squash the commits? Then I'll merge it.

**Comment by xcompass** [comment]
@bpickard saw your merged branch. But this PR is still not squashed. Maybe you did something wrong. Let me know if you need help to squash them.

**Comment by bpickard** [comment]
Thanks, @xcompass, I will need some help then. Richard helped me squash the commits into a single one but I'm guessing there's something else I'm missing.

**Comment by richardtape** [comment]
Bill, completely my fault. We shouldn't have made another branch.

Ensure your master is up to date, then check out your normal feature 
branch and go back to before we made the ill-fated merge with the squash 
branch, then

`git rebase -i master`

Change the commit message as before (i.e. leave the first one as 'pick' 
and then all the other 'pick's to 'squash')

Then you need to *force* push (push with the `-f` flag) as you're 
"rewriting history"

My bad, it's been a while since I've squashed me some commits :)



bpickard wrote:

**Comment by bpickard** [comment]
Okay thanks @richardtape, I think I've done it correctly now.

**Pull request merged**